extensions/core/runtime/src/main/java/io/quarkiverse/cxf/CxfClientProducer.java

```diff
@@ -94,7 +101,7 @@ private Object produceCxfClient(CXFClientInfo cxfClientInfo) {
         QuarkusClientFactoryBean quarkusClientFactoryBean = new QuarkusClientFactoryBean();
         QuarkusJaxWsProxyFactoryBean factory = new QuarkusJaxWsProxyFactoryBean(quarkusClientFactoryBean, interfaces);
         factory.setServiceClass(seiClass);
-        LOGGER.debugf("using servicename {%s}%s", cxfClientInfo.getWsNamespace(), cxfClientInfo.getWsName());
+        LOGGER.warnf("using servicename {%s}%s", cxfClientInfo.getWsNamespace(), cxfClientInfo.getWsName());
```

Comment:
@ppalaga just noticed this output in 2.4.0 - looks like an oversight?

Reply:
See #1029

Reply:
Thanks, @famod! It is definitely a left over from my experiments.